batch/sql/add-pool-label.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `pools` ADD `label` VARCHAR(100) NOT NULL;
```

Comment:
Because it's `NOT NULL`, how would it work with other pools, e.g. https://github.com/populationgenomics/hail/blob/ac1e946c13108e342f470a29513f8a3af01cf04f/batch/sql/add-nonpreemptible-pools.sql#L28-L30 doesn't set a value for `label`

Reply:
It adds the empty string as a value, which then ends up selecting the default pools when not specifying a pool label for a job.

I actually used `NULL` values initially, but that gets really tricky when updating the label via the HTML forms. Using the empty string for clearing the label was much easier.